frinx-uniconfig/user-guide/network-management-protocols/uniconfig_gnmi/readme.md

```diff
@@ -0,0 +1,213 @@
+(once the schema context is built, it is cached for next runs) and all necessary services are created (DOMDataBroker, 
+DOMRpcService, DOMNotification/Subscription service). After all of this is done, gNMI southbound plugin will mark 
+the connection status of the particular device as READY, so UniConfig layer can handle it.
+
+## Important features
+
+When gNMI device is installed, it is possible to do:
+
+- read (gNMI GET), create/update/delete (gNMI SET) the configuration of device:
+  - via UniConfig topology
+  - via gNMI souhtbound topology (using `yang-ext:mount`)
+  - via yang-patch operations over UniConfig/gNMI souhtbound topology (using `yang-ext:mount`)
+- manipulate with a device via gNOI RPCs
+- use gNMI SET update operation for specific paths
+- use gNMI SET replace operation for specific paths
+- order config changes (replace/delete operation) according to dependency paths
+- remove module-name from list keys for specific paths
+- read from ALL datastore for specific paths
+- subscribe to telemetry streaming for specific paths (gNMI Subscribe)
+- store failed installations
+- enable logging-broker for gNMI messages (SET/GET)
+- use DryRunCommit feature
+
+### gNOI 
+
+The gNOI (gRPC Network Operations Interface) defines gRPC-based microservices 
+for executing operational commands on network devices. By default gNMI southbound plugin is capable of 
+executing Certification, File, System and OS RPCs according to protobuf files <https://github.com/openconfig/gnoi>. 
+Apart from the common ones, it is possible to invoke also SONiC type gNOI RPCs by specifying 
+`"gnmi-topology:device-type" : "sonic"` in install request inside of `connection-parameters` container. 
+It's necessary to have `gnoi-sonic@2023-03-17.yang` schema model in cache directory for this specific node, 
+to be able to invoke it via RESTCONF. 
+
+Example of gNOI RPC invocation
+------------------------------
+
+```bash Invocation of SONiC specific copy-config RPC
+curl --location --request POST 'http://localhost:8181/rests/operations/network-topology:network-topology/topology=gnmi-topology/node=sonic/yang-ext:mount/gnoi-sonic:copy-config' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+        "input": {
+            "input": {
+                "source": "running-configuration",
+                "destination": "startup-configuration"
+            }
+        }
+    }'
+```
+
+```json Response
+{
+  "output": {
+    "output": {
+        "status_detail": "SUCCESS."
+    }
+  }
+}
+```
+
+### Update paths
+
+Update paths feature tells gNMI southbound plugin to process the intended config 
+changes (calculated from UniConfig Diff) as gNMI SET message - Update operation. 
+The gNMI southbound plugin basically checks for config changes paths if they are relative 
+to those specified in the install request. The main purpose of using update paths feature 
```

Comment:
"checks for config changes paths if they are relative to those specified in the install request"

What does this mean exactly? Is there another way to phrase it?

Reply:
rephrased